Further clarified the option of using built-in labels while being able to label non-Office files with the AIP app (#6585)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/sensitivity-labels-office-apps.md

### Office built-in labeling client and the Azure Information Protection client

> You may want to use the built-in sensitivity labeling capabilities in Office to make use of the latest features and enhance user experience, while at the same time having the option of labeling and protecting non-Office files from the desktop with the Azure Information Protection client. 

Reviewer: ```suggestion
You might want to use the built-in sensitivity labeling capabilities in Office to make use of the latest features and enhance user experience, while at the same time having the option of labeling and protecting non-Office files from the desktop with the Azure Information Protection client. 
```

> By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2) the built-in labeling client is turned off in their Office apps and the plugin functionality is used instead. 

Reviewer: ```suggestion
By default, if users have the [Azure Information Protection client installed](/azure/information-protection/rms-client/aip-clientv2), the built-in labeling client is turned off in their Office apps and the plug-in functionality is used instead. 
```

> To maintain the ability to use the Azure Information Protection clientfor labeling non-Office files from the desktop while using the built-in labeling functinality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).

Reviewer: ```suggestion
To maintain the ability to use the Azure Information Protection client for labeling non-Office files from the desktop while using the built-in labeling functionality within Office apps, we recommend you use the Group Policy setting **List of managed add-ins** as documented in [No Add-ins loaded due to group policy settings for Office 2013 and Office 2016 programs](https://support.microsoft.com/help/2733070/no-add-ins-loaded-due-to-group-policy-settings-for-office-2013-and-off).
```

Deploy this setting by using Group Policy, or by using the [Office cloud policy service](/DeployOffice/overview-office-cloud-policy-service).

> Alternatively, the following registry value can be used to quickly toggle between using the Azure Information Protection plugin and built-in sensitivity labels in Office apps:

Reviewer: ```suggestion
Alternatively, the following registry value can be used to quickly toggle between using the Azure Information Protection plug-in and built-in sensitivity labels in Office apps:
```

|**Registry key**|**Type**|**Value** |
|:-------|:------:|--------:|
|HKCU\Software\Microsoft\Office\16.0\Common\Security\Labels\UseOfficeForLabelling  |REG_DWORD  |1  |

> Setting this value to 1 enables the built-in sensitivity labeling options in Office, setting it to 0 enables the AIP plugin if it is installed. 

Reviewer: ```suggestion
Setting this value to **1** enables the built-in sensitivity labeling options in Office; setting it to **0** enables the AIP plug-in if it is installed. 
```
